brandingbrand/flagship
# chore: bump all package versions and setup docs to release

## Description
update docs and change versions for release of versions code-cli 12.0.0, code-core 12.0.0, and all plugins as version 1.0.0
update peer dependencies to allow versions to work together

## Test
* run docs `yarn dev:docs`
* verify looks good
